boards/renesas/ek_ra2a1/doc/index.rst

```diff
@@ -0,0 +1,105 @@
+.. _ek_ra2a1:
+
+EK-RA2A1
```

Comment:
I applied all of your comments about the documentation.

I didn't include any photos or illustrations because I wasn't clear about the copyright permissions. If you can update, I hope you to do so.

Reply:
I am sorry for my late. I have no comment for your PR. For the photos of RA2A1, we will update it in https://github.com/zephyrproject-rtos/zephyr/pull/77298 after this PR is merged.